diff --git a/agent/agent.c b/agent/agent.c
index fe7a299..becfb69 100644
--- a/agent/agent.c
+++ b/agent/agent.c
@@ -77,7 +77,7 @@
 
 #define DEFAULT_STUN_PORT  3478
 #define DEFAULT_UPNP_TIMEOUT 200  /* milliseconds */
-#define DEFAULT_IDLE_TIMEOUT 5000 /* milliseconds */
+#define DEFAULT_IDLE_TIMEOUT 60000 /* milliseconds */
 
 #define MAX_TCP_MTU 1400 /* Use 1400 because of VPNs and we assume IEE 802.3 */
 
